### DISCUSSION TITLE
link between users and clients

### DISCUSSION THREAD

**Discussion by marcobustaffa** [JanssenProject/jans#5606] [Q&A] [answered]
Hello there,
I was wondering if is it possible to associate users to a specific openid client so that any user that is not specifically linked to the client can't access the resource.

Thanks

**Comment by ossdhaval** [comment, marked as answer]
Hi @Mallow00. Welcome to Janssen Project.

See if this document on [multi-tenancy](https://docs.jans.io/v1.0.15/admin/planning/multi-tenancy/) can help with your use-case. And I think it'll be worth having a look at RBAC policy implementations as well.

**Reply by marcobustaffa** [reply]
Thank you @ossdhaval, I think this is what I was looking for.

**Comment by nynymike** [comment]
Maybe if the client did DCR. See this [proposed design](https://github.com/JanssenProject/jans/wiki/Mobile-DPoP-FIDO-Authn).
